### DISCUSSION TITLE
mspider/intro/comp-archi-ui/cp-component

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2916] [Announcements]
# mspider/intro/comp-archi-ui/cp-component

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/comp-archi-ui/cp-component.html

**Comment by nicole-lihui** [comment]
mspider 组件缺少一个 mspider-work-api-service (服务管理，isito 策略相关istio 控制资源的 API操作)

mspider-api-service  主要对网格管理，网格网关，边车等具体实例管理能力 的 API 操作
